geo/src/test/java/org/odk/collect/geo/selection/SelectionMapFragmentTest.kt

```diff
@@ -804,6 +804,75 @@ class SelectionMapFragmentTest {
         scenario.moveToState(Lifecycle.State.DESTROYED)
     }
 
+    @Test // https://github.com/getodk/collect/issues/5540
```

Comment:
I don't think we need to link the Github issue in either of these tests. The behaviour makes a lot of sense without the extra context.